core/orchestrator.go

```diff
@@ -79,6 +80,40 @@ func (orch *orchestrator) TranscoderResults(tcId int64, res *RemoteTranscoderRes
 	orch.node.transcoderResults(tcId, res)
 }
 
+func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID ManifestID) error {
```

Comment:
I think we'll get a nil pointer exception on L94 if the node is running in off-chain mode since `orch.node.Recipient` will be nil. The rest of the orchestrator methods that depend on the presence of a ETH client first check that both the node and the ETH client are not nil. Perhaps we can add a similar check here:

```
if orch.node == nil || orch.node.Recipient == nil {
    return nil
}
```

Reply:
Awesome tip, thank you! I had that thought for a moment but then felt like it might be too defensive, forgetting all about off-chain mode! My bad!